src/main/flight/mixer.h

```diff
@@ -66,6 +66,7 @@ typedef struct mixer_t {
 typedef struct mixerConfig_s {
     uint8_t pid_at_min_throttle;            // when enabled pids are used at minimum throttle
     int8_t yaw_direction;
+    int8_t yaw_jump_prevention_limit;       // make limit configurable (original fixed value was 100)
```

Comment:
IMO we don't need super fine resolution on the limit, so I would use uint8_t and just multiply it by 4 or something. That'd give us up to 1020 as a cap which is a huge correction (the mixer effectively uses the PID sum as a PWM value to add to the motor outputs, so 1020 corresponds to more than the available range of motor commands anyway).